### PR TITLE
Heatmap colorbar

### PR DESCRIPTION
**Pull request by mortonjt**
This also depends on #214 
 
Essentially, this adds in a colorbar for the heatmap. 

screen shot 2017-07-17 at 6 05 38 pm

We also allow for keyword arguments to be directly passed into the heatmap, so that different color normalization schemes can be used (for instance).

### CODE REVIEW

**Comment by mortonjt** [comment]
Alright - we got the tests passing!  @qiyunzhu can you sanity check this?  You already saw the output in q2-gneiss -- so this should be a very fast review.

**Comment by mortonjt** [comment]
@qiyunzhu @antgonza can you guys please review / merge this?

File: gneiss/plot/_heatmap.py [hunk 1]
```diff
@@ -48,7 +48,8 @@ def heatmap(table, tree, mdvar, highlights=None, cmap='viridis',
         (default=2)
     figsize: tuple of int
         Species (width, height) for figure. (default=(5, 5))
-
+    **kwargs : dict
+        Arguments to pass into the heatmap plotting.
```
Comment (qiyunzhu) [inline gneiss/plot/_heatmap.py]: Be consistent with "Arguments to pass into" and "Arguments to be passed into".

Comment (mortonjt) [inline gneiss/plot/_heatmap.py]: Thanks!!

**Comment by mortonjt** [comment]
Alright - since this is approved and we need to push out a minor release now, I'm going to merge this.

**Pull request merged**